projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -115,35 +115,7 @@ public function get_preferred_view( $screen, $fallback_global_preference = true
 	 * Adds Plugins menu.
 	 */
 	public function add_plugins_menu() {
-		global $submenu;
-		if (
-			isset( $submenu['plugins.php'] )
-			/**
-			 * Whether to enable the marketplace feature entrypoint.
-			 * This filter is specific to WPCOM, that's why there is no
-			 * need to use `jetpack_` prefix.
-			 *
-			 * @use add_filter( 'wpcom_marketplace_enabled', '__return_true' );
-			 * @module masterbar
-			 * @since 10.3
-			 * @param bool $wpcom_marketplace_enabled Load the WordPress.com Marketplace feature. Default to false.
-			 */
-			&& apply_filters( 'wpcom_marketplace_enabled', false )
-		) {
-			$plugins_submenu = $submenu['plugins.php'];
-			$slug_to_update  = 'plugin-install.php';
-
-			// Move "Add New" plugin submenu ( `plugin-install.php` ) to the top position.
-			foreach ( $plugins_submenu as $submenu_key => $submenu_keys ) {
-				if ( $submenu_keys[2] === $slug_to_update ) {
-					// phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
-					$submenu['plugins.php'] = array( $submenu_key => $plugins_submenu[ $submenu_key ] ) + $plugins_submenu;
-				}
-			}
-
-			$submenus_to_update = array( $slug_to_update => 'https://wordpress.com/plugins/' . $this->domain );
-			$this->update_submenus( 'plugins.php', $submenus_to_update );
-		}
+		// Keep empty body to force WordPress core Plugins menu.
```

Comment:
As @obenland [pointed out](https://github.com/Automattic/jetpack/pull/22786#discussion_r806079617), since this class is meant to be Atomic-specific and `class-atomic-admin-menu.php` only runs on Atomic, adding a logic here that is later overridden in `wpcomsh` feels unnecessary.

Can we please move the changes from 811-gh-Automattic/wpcomsh here and remove that PR?

Reply:
Oh, I understood it other way around. I will quickly make those changes.